src/test/test-capability.c

```diff
@@ -20,8 +20,13 @@
 static uid_t test_uid = -1;
 static gid_t test_gid = -1;
 
-/* We keep CAP_DAC_OVERRIDE to avoid errors with gcov when doing test coverage */
-static uint64_t test_flags = 1ULL << CAP_DAC_OVERRIDE;
+#ifdef __SANITIZE_ADDRESS__
+        /* Keep CAP_SYS_PTRACE when running under Address Sanitizer */
+        static uint64_t test_flags = 1ULL << CAP_SYS_PTRACE;
```

Comment:
we usually do not add an indentation level when using `#ifdef` (except sometimes for pre-processor expressions themselves). hence, please don't indent this here...

Reply:
I knew there was something off with this formatting, thanks

Reply:
To judge from this and my failed attempt to run two jobs in parallel via the `.yml` file, It seems I have been having trouble indenting text this week :-) Thank you!